CMakeLists.txt

```diff
@@ -101,6 +101,7 @@ option(SSG_PRODUCT_OPENSUSE "If enabled, the openSUSE SCAP content will be built
 option(SSG_PRODUCT_RHEL7 "If enabled, the RHEL7 SCAP content will be built" ${SSG_PRODUCT_DEFAULT})
 option(SSG_PRODUCT_RHEL8 "If enabled, the RHEL8 SCAP content will be built" ${SSG_PRODUCT_DEFAULT})
 option(SSG_PRODUCT_RHEL9 "If enabled, the RHEL9 SCAP content will be built" ${SSG_PRODUCT_DEFAULT})
+option(SSG_PRODUCT_RHEL10 "If enabled, the RHEL10 SCAP content will be built" FALSE)
```

Comment:
We want to start builidng it by default. Also we want to test the RHEL 10 content in GitHub CI, gating jobs.

Reply:
while that's true, I wonder when this will become feasible since there are no releases of such version so far, maybe I'm wrong... but that was one way to prevent people from trying to accidentally use RHEL10 content

Reply:
I would say that there is not too much to test since the product does not exist yet. I would keep it disabled by default and save resources. We can build it on-demand for now if we want to experiment something. At some point in the future it will make more sense to enable it by default and this will be easy since we already have this structure created.

Reply:
Keep in mind that RHEL10 will only be built when you run `./build_product` without specifing a product or `cmake ..`.